view/perspective/perspective.pug

```diff
@@ -21,19 +21,19 @@ html(lang = 'en')
     div#lens_loading_spinner
       .slds-spinner_container
         .slds-spinner--brand.slds-spinner.slds-spinner--medium(role='alert')
-            span.slds-assistive-text Loading
-            |     
-            .slds-spinner__dot-a
-            |     
-            .slds-spinner__dot-b
+          span.slds-assistive-text Loading
+          .slds-spinner__dot-a
+          .slds-spinner__dot-b
```

Comment:
Some time back I changed this pug to something like this but I was not able to load any perspective after this change. Wondering if the perspective still load after this change.